server.js

```diff
@@ -54,12 +54,14 @@ app.use('/', auth, cors(), morgan('combined'), graphqlHTTP(request => {
   loaders.clearAll();
 
   const accessToken = request.headers['x-access-token'];
+  const userID = request.headers['x-user-id'];
```

Comment:
Neat, so should we update web stack middleware (i think https://github.com/artsy/artsy-passport) to start setting this?